gradebookng/tool/src/java/org/sakaiproject/gradebookng/business/GradebookNgBusinessService.java

```diff
@@ -515,6 +518,38 @@ private Gradebook getGradebook(final String siteId) {
 		return rval;
 	}
 
+	/**
+	* Retrieve the categories visible to the given student.
+	*
+	* This should only be called if you are wanting to view the categories that a student would see (ie if you ARE a student, or if you
+	* are an instructor using the student review mode)
+	*
+	* @param studentUuid
+	* @return list of visible categories
+	*/
+	public List<CategoryDefinition> getGradebookCategoriesForStudent(String studentUuid) {
```

Comment:
Are we certain we want students knowing details of the category setup and scores that are dropped etc?

Reply:
@kyleblythe  thoughts?

Reply:
We don't hide released grades from students so all final grade calcs can be reverse engineered. I don't see an argument to support being opaque about what items are dropped.

Reply:
Gradebook Classic shows the dropped items to students.